Policies/github-privacy-statement.md

```diff
@@ -33,13 +34,13 @@ To see our Privacy Notice to residents of California, please go to [GitHub's Not
 
 | Section | What can you find there? |
 |---|---|
-| [Who is responsible for the processing of your information](#who-is-responsible-for-the-processing-of-your-information) | Subject to limited exceptions, GitHub is the controller and entity responsible for the processing of your Personal Data in connection with the Website or Service. |
```

Comment:
you sure

Reply:
They definitely are certain. They would not have modified this as they have alternatively.